src/Uno.UWP/UI/Colors.cs

```diff
@@ -250,10 +250,9 @@ public static Color FromARGB(string colorCode)
 			}
 			else
 			{
-				r = 0xFF;
-				g = 0x0;
-				b = 0x0;
+				throw new ArgumentException($"Cannot parse color '{colorCode}'.");
 			}
+
```

Comment:
@spouliot maybe that's the red we were looking for ? :)

Reply:
I'll check that out asap!

Reply:
No, the pinkish issue does not hit that code path 😢 

And now I realize I opened the original issue... no wonder it ringed a bell (but not loud enough it seems).